.rubocop.yml

```diff
@@ -3,3 +3,7 @@ require:
 
 AllCops:
   TargetRubyVersion: 2.5
+
+# Commonly used screens these days easily fit more than 80 characters.
```

Comment:
Definitely! I think This metric is not so much about what technology offers but about human readability and maintainability. Reading code vertically (easier with less characters per line) produces less cognitive overhead for the reader (https://baymard.com/blog/line-length-readability). I don't mind 120 characters, esp. in projects with long namespaces. I take technically out-dated but I also don't see the benefit clearly here. I guess this is just a matter of taste.

Reply:
This rule mainly applies to reading normal text I think, code is slightly different, mainly because the more code you see the better context you have by making just a glance. I'd personally prefer longer lines than forcefully broken into many short lines, which sometimes results in a single function call covering 1/5th of the screen. For text on the other hand, I prefer very narrow paragraphs - what most aware web portals present :) With code it's really the eternal struggle: spaces or tabs, long or narrow lines. Someone always has to make a call...